app/constants/offer-sorting-options.js

```diff
@@ -0,0 +1,30 @@
+export const offerSortingOptions = [
```

Comment:
Nice job extracting a config, makes me wonder whether the `constants` would be better named as `config` or something like that